src/HeaderData.php

```diff
@@ -83,6 +83,7 @@ public function parseHeader($key) {
             switch ($key) {
                 // Parse Audience header.
                 case 'Audience':
+                case 'Audience-Set':
```

Comment:
This looks great! Mind adding a small test for it too? Thanks Chris!

Reply:
👍 Added in https://github.com/pantheon-systems/pantheon-edge-integrations/pull/13/commits/b7ed484dba41fbcccbaeb977648aeb6d41d7c3e0